src/modules/Security.rb

```diff
@@ -730,6 +730,10 @@ def Import(settings)
     # (For use by autoinstallation.)
     # @return [Hash] Dumped settings (later acceptable by Import ())
     def Export
```

Comment:
For me, it is somehow strange that Export modify the internal state of Security module. It should only export it.